CMakeLists.txt

```diff
@@ -10,6 +10,18 @@ if(NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE Release)
 endif()
 
+find_package(Git REQUIRED)
```

Comment:
How about not making it required, and putting the rest of the block in a "if git found"? This would break the build for people that don't have git marked as build dependency

Reply:
Done.